src/api-state/index.js

```diff
@@ -1,8 +1,23 @@
 import { combineEpics, createEpicMiddleware } from 'redux-observable';
 
-import getSyncEpic from '../epics/sync';
-import getCacheEpic from '../epics/cache';
-import { postEpic, deleteEpic } from '../epics/mutate'; // DEPRECATED
+import getSyncEpic from './sync';
+import getCacheEpic from './cache';
+import { postEpic, deleteEpic } from './mutate'; // DEPRECATED
+
+// export specific values of internal modules
+export {
```

Comment:
yes - the `index.js` in the root folder for the module is effectively the 'public interface' for the module. The expectation is that dependent modules will _only_ use `require('./src/api-state')` or `import ___ from './src/api-state'` - importing from deeper modules is discouraged and undocumented, although still technically possible because JS.

This `src/api-state/index.js` module will eventually be published to npm as `mwp-api-state`, so imports will be simple/standard.

The _major_ exception to this pattern is when you want to export a bunch of things that are designed to be used independently, like MWC. Using an `index.js` public interface means that each time you import one thing, you have to bundle _everything_ - this can be particularly problematic when some of the exports are designed to be used exclusively on the server, which often has very large third-party dependencies.

For something like `api-state`, that's not a problem because it's designed to be used all-or-nothing, server side and browser side.